helper.d.ts

```diff
@@ -1,8 +1,12 @@
 import fastify from 'fastify'
 
+type BuildOptions = {
+    skipOverride?: boolean
+}
```

Comment:
I can think of other scenarios where this might be useful, such as the `beforeReady` callback which could allow decorators to be overridden, e.g. to make a method fail.